ExtraEnemyCustomization/Customizations/Abilities/AttackCustoms/BleedAttackCustom.cs

```diff
@@ -32,13 +32,7 @@ public override void OnConfigUnloaded()
 
         protected override void OnApplyEffect(BleedSetting setting, PlayerAgent player, EnemyAgent inflictor)
         {
-            if (inflictor.TryCastToEnemyAgent(out var enemy))
-            {
-                if (IsTarget(enemy))
-                {
-                    setting.DoBleed(player);
-                }
-            }
```

Comment:
Just out of curiosity, why were these checks removed?

Reply:
its already done by AttackCustomBase